docs/content/docs/operations/helm.md

```diff
@@ -74,7 +74,8 @@ The configurable parameters of the Helm chart and which default values as detail
 | operatorPod.dnsPolicy | DNS policy to be used by the operator pod. | |
 | operatorPod.dnsConfig | DNS configuration to be used by the operator pod. | |
 | operatorPod.nodeSelector | Custom nodeSelector to be added to the operator pod. | |
-| operatorPod.resources | Custom resources block to be added to the operator pod. | |
+| operatorPod.resources | Custom resources block to be added to the operator pod on main container. | |
+| operatorPod.flinkWebhookResources | Custom resources block to be added to the operator pod on flink-webhook container. | |
```

Comment:
How about `webhook.resurces` instead? It is a bit more concise and would make sense to group it like that?

Reply:
Maybe there should be new key `webhookPod` to be consistent with `operatorPod`? Or jus use `operatorPod.resources` as a some configuration for webhook container is used from `operatorPod` key?

Reply:
I vote for `webhookPod` for the sake of consistency

Reply:
Should `webhookPod.dnsConfig`be created for consistency, as currently webhook container config uses `operatorPod.dnsConfig`?

Reply:
Let us do `webhookPod` then and include the `dnsConfig` please.

Reply:
cc @Vince-Chenal

Reply:
I moved it to `.Values.operatorPod.webhook.resources`, does it sound ok for you ?